x/cash/dynamicfee.go

```diff
@@ -65,15 +67,14 @@ func (d DynamicFeeDecorator) Check(ctx weave.Context, store weave.KVStore, tx we
 		return res, err
 	}
 
-	// read config
-	minFee := d.getMinFee(store)
+	minFee := gconf.Coin(store, GconfMinimalFee)
```

Comment:
Can you explain why this is undone?
gconf hits the db each time, right? (storing in some global lookup table would break all tests)

Reply:
Discussed in https://github.com/iov-one/weave/pull/359#discussion_r260680595